src/components/ens-registration/SearchInput/SearchInput.tsx

```diff
@@ -128,6 +128,7 @@ const SearchInput = ({
             </Column>
             <Input
               autoCorrect={false}
+              autoFocus
```

Comment:
We use the `useMagicAutofocus` hook here as well. Do we need this?

Reply:
No. Sorry, we don't 😄